M2/Macaulay2/packages/Msolve.m2

```diff
@@ -44,7 +44,7 @@ msolveProgram = findProgram("msolve", "msolve --help",
 if msolveProgram === null then (
     if not (options currentPackage).OptionalComponentsPresent
```

Comment:
I don't think this will ever be true now that `OptionalComponentsPresent` stuff was removed.

Reply:
Wait, we haven't merged that PR yet!

Reply:
Oh I see it was removed from this package's headers. Yeah you're right